app/src/layouts/appLayout/appSidebar/appSidebar.jsx

```diff
@@ -100,7 +107,7 @@ export class AppSidebar extends Component {
         message: <FormattedMessage id={'Sidebar.dashboardsBtn'} defaultMessage={'Dashboards'} />,
       },
       {
-        onClick: () => this.onClickButton(SIDEBAR_EVENTS.CLICK_LAUNCHES_BTN),
+        onClick: () => this.onClickButton(SIDEBAR_EVENTS.clickLaunchesBtn(this.getFilterCount())),
```

Comment:
Why not just like `.clickLaunchesBtn(this.props.launchFilters.length)`?
Should we count unsaved filters?